src/test-util/compare-dom.js

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
I've skimmed this and it seems fine at first glance. I feel like a light review is probably OK because these are "just" test affordances, but let me know if you'd like something more for-real.